src/components/Shows/styles.scss

```diff
@@ -0,0 +1,8 @@
+@import 'variables.scss';
```

Comment:
Same with `variables.scss` here as in the previous comment. It only increases the file size when fetching the CSS :)